README.md

```diff
@@ -1,20 +1,18 @@
+Ghidra is a software reverse engineering (SRE) framework created and maintained by the [National Security Agency][nsa] Research Directorate. This framework includes a suite of full-featured, high-end software analysis tools that enable users to analyze compiled code on a variety of platforms including Windows, Mac OS, and Linux. Capabilities include disassembly, assembly, decompilation, graphing, and scripting, along with hundreds of other features. Ghidra supports a wide variety of process instruction sets and executable formats and can be run in both user-interactive and automated modes. Users can also develop their own Ghidra plug-in components and/or scripts using Java or Python.
 
-In support of NSA's Cybersecurity mission, Ghidra was built to solve scaling and teaming problems on complex SRE efforts, and to provide a customizable and extensible SRE research platform. NSA has applied Ghidra SRE capabilities to a variety of problems that involve analyzing malicious code and generating deep insights for SRE analysts who seek a better understanding of potential vulnerabilities in networks and systems.
+In support of the NSA's Cybersecurity mission, Ghidra was built to allow scaling and collaboration on complex SRE efforts, and to provide a customizable and extensible SRE research platform. The NSA has applied Ghidra SRE to a variety of problems that involve analyzing malicious code and generating deep insights for SRE analysts who seek a better understanding of potential vulnerabilities in networks and systems.
 
-This repository is a placeholder for the full open source release.
-Be assured efforts are under way to make the software available here.
+This repository is a placeholder for the full open-source release.
 In the meantime, enjoy using Ghidra on your SRE efforts, developing your
 own scripts and plugins, and perusing the over-one-million-lines of Java and
 Sleigh code released within the initial public release.
 The release can be downloaded from our [project homepage][project].
-Please consider taking a look at our [contributor guide][contrib] to see how
-you can participate in this open source project when it becomes available.
+Please take a look at our [contributor guide][contrib] to learn how
+you can participate in this open-source project when the code becomes available.
 
-If you are interested in projects like this and would like to develop this, and
-other cybersecurity tools, for NSA to help protect our nation and its allies,
+If you are interested in contributing to cybersecurity projects like this and to help the NSA protect the United States of America and its allies,
```

Comment:
There is only one NSA. It only has one nation it protects which is “ours.” I think saying “help NSA protect the United States of America” is like saying “I’m a huge Dallas Cowboys American Football fan.” It’s unnecessarily specific.

Reply:
Maybe you should look up the meaning of "www". Github is not a branch of the US government. When communicating (I mean with someone other than yourself), it's usually necessary to provide context. For example, "football" and "world champions" have very different meanings inside and outside the US.